fragments/node-boot.sh

```diff
@@ -181,10 +181,13 @@ then
     systemd_add_docker_socket
 fi
 
-docker_check_for_storage_device $DOCKER_VOLUME_DEVICE
+if [ -n "$VOLUME_ID" ]
+then
+    docker_check_for_storage_device $DOCKER_VOLUME_DEVICE
 
-# configure the external docker volume for LVM management
-docker_set_storage_device $DOCKER_VOLUME_DEVICE
+    # configure the external docker volume for LVM management
+    docker_set_storage_device $DOCKER_VOLUME_DEVICE
+fi
```

Comment:
Few lines bellow is duplicated:
/usr/bin/docker-storage-setup || notify_failure "Docker Storage setup failed"

I guess it's an old forgotten fragment - could you please remove it?